Configuration/build.gradle

```diff
@@ -16,6 +19,7 @@ dependencies {
     annotationProcessor libs.autoservice.compiler
 
     testImplementation project(path: ':Base', configuration: 'tests')
+    testImplementation libs.commons.compress
```

Comment:
This seems surprising. Weird thing to only need for test...

Reply:
Some of the tests depend on IOUtils:

```
> Task :Configuration:compileTestJava FAILED
/home/devin/dev/deephaven/deephaven-core/Configuration/src/test/java/io/deephaven/configuration/PropertyInputStreamLoaderTraditionalTest.java:14: error: package org.apache.commons.compress.utils does not exist
import org.apache.commons.compress.utils.IOUtils;
                                        ^
/home/devin/dev/deephaven/deephaven-core/Configuration/src/test/java/io/deephaven/configuration/PropertyInputStreamLoaderTraditionalTest.java:31: error: cannot find symbol
            bytes = IOUtils.toByteArray(in);
                    ^
  symbol:   variable IOUtils
  location: class PropertyInputStreamLoaderTraditionalTest
/home/devin/dev/deephaven/deephaven-core/Configuration/src/test/java/io/deephaven/configuration/PropertyInputStreamLoaderTraditionalTest.java:43: error: cannot find symbol
            bytes = IOUtils.toByteArray(in);
                    ^
  symbol:   variable IOUtils
  location: class PropertyInputStreamLoaderTraditionalTest
3 errors
```